### PR TITLE
Review OS detection. Ram pick out. Improvment on the code

### LINKED ISSUE #68: Review OS detection
SuSE hosts show a warning when we try to determine the OS version:

```
OS Version: SuSE SUSE Linux Enterprise Server 12 (x86_64) VERSION = 12 PATCHLEVEL = 3 # This file is deprecated and will be removed in a future service pack or release. # Please check /etc/os-release for details about this release.
```

We should review this part of the script and maybe switch to a more future-ready version of OS detection

### PR DESCRIPTION
**Pull request by afeefghannam89**
fixes #68

### CODE REVIEW

File: icinga-diagnostics.sh [hunk 1]
```diff
@@ -425,7 +401,7 @@ doc_os() {
       echo -n "RAM: "
       if [ "${OS}" = "SuSE" ]
       then
-        free -g | grep ^Mem | awk '{print $2"G"}'
+        free - g | grep Mem | awk '{print $2 / 1024"MB"}'
```
Comment (dnsmichi) [inline icinga-diagnostics.sh]: Is that space intended?

Comment (afeefghannam89) [reply icinga-diagnostics.sh]: no i did not notice it

**Comment by afeefghannam89** [comment]
okay i will do it :)

File: icinga-diagnostics.sh (hunk 1 above)
Comment (widhalmt) [reply icinga-diagnostics.sh]: The one in "- g"? Nope.

**Comment by widhalmt** [comment]
Thank you very much for you improvements!

**Pull request merged**